### ISSUE TITLE
Normalizing TFIDF & Counting steps

### ISSUE THREAD

**Issue by ArdalanM** [beniz/deepdetect#82]
Since we are dealing with nets, it would be great to have the possibility to normalize the feature creation step. 

Just like sklearn we could have options like:

- norm : ‘l1’, ‘l2’: Norm used to normalize term vectors.
- smooth_idf : Smooth idf weights by adding one to document frequencies

source: http://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html
